internal/plan/plan_test.go

```diff
@@ -1017,3 +1046,135 @@ func (s *S) TestMarshalLayer(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(string(out), Equals, string(layerBytes))
 }
+
+var svcArgsTests = []struct {
```

Comment:
I think these tests will become significantly simpler when moved out of plan. I think we can mostly test the `GetCommand()` (or whatever we call it) function in isolation as a pure function.

Reply:
I added the new tests, please take a look.